<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAFailProfilePulseWgt</class>
 <widget class="QWidget" name="GTAFailProfilePulseWgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>duty Cycle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dutyCycleLE"/>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_16">
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Phase</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phaseLE"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_14">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Offset     </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="offsetLE"/>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_17">
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>period</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="periodLE"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
